mkdocs/docs/api.md

```diff
@@ -700,6 +700,137 @@ parent_id: [[null,4358109269873137077,null,4358109269873137077]]
 is_current_ancestor: [[true,false,true,true]]
 ```
 
+### Files
+
+Inspect the data files in the current snapshot of the table:
+
+```python
+table.inspect.files()
+```
+
+```
+pyarrow.Table
```

Comment:
looks like the current schema doesn't match the comment here

Reply:
Sorry I am unable to catch that, what is the schema discrepancy?

Reply:
Steps to reproduce: 
add a breakpoint [here](https://github.com/apache/iceberg-python/pull/614/files#diff-f713e80c8536d27e9acdf3bddae9e3a447a36aa96d857c7c3c114a494570dc4fR659)

Run in terminal
```
poetry run pytest tests/integration/test_inspect_table.py -v -m integration -k test_inspect_files -s -vvv
``` 

Run in debugger
```
df.schema
```

Result
```
(Pdb) df.schema
content: int8 not null
file_path: string not null
file_format: dictionary<values=string, indices=int32, ordered=0> not null
spec_id: int32 not null
record_count: int64 not null
file_size_in_bytes: int64 not null
column_sizes: map<int32, int64>
  child 0, entries: struct<key: int32 not null, value: int64> not null
      child 0, key: int32 not null
      child 1, value: int64
value_counts: map<int32, int64>
  child 0, entries: struct<key: int32 not null, value: int64> not null
      child 0, key: int32 not null
      child 1, value: int64
null_value_counts: map<int32, int64>
  child 0, entries: struct<key: int32 not null, value: int64> not null
      child 0, key: int32 not null
      child 1, value: int64
nan_value_counts: map<int32, int64>
  child 0, entries: struct<key: int32 not null, value: int64> not null
      child 0, key: int32 not null
      child 1, value: int64
lower_bounds: map<int32, binary>
  child 0, entries: struct<key: int32 not null, value: binary> not null
      child 0, key: int32 not null
      child 1, value: binary
upper_bounds: map<int32, binary>
  child 0, entries: struct<key: int32 not null, value: binary> not null
      child 0, key: int32 not null
      child 1, value: binary
key_metadata: binary
split_offsets: list<item: int64>
  child 0, item: int64
equality_ids: list<item: int32>
  child 0, item: int32
sort_order_id: int32
readable_metrics: struct<bool: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: bool, upper_bound: bool> not null, string: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: large_string, upper_bound: large_string> not null, string_long: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: large_string, upper_bound: large_string> not null, int: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: int32, upper_bound: int32> not null, long: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: int64, upper_bound: int64> not null, float: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: float, upper_bound: float> not null, double: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: double, upper_bound: double> not null, timestamp: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: timestamp[us], upper_bound: timestamp[us]> not null, timestamptz: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: timestamp[us, tz=UTC], upper_bound: timestamp[us, tz=UTC]> not null, date: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: date32[day], upper_bound: date32[day]> not null, binary: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: large_binary, upper_bound: large_binary> not null, fixed: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: fixed_size_binary[16], upper_bound: fixed_size_binary[16]> not null>
  child 0, bool: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: bool, upper_bound: bool> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: bool
      child 5, upper_bound: bool
  child 1, string: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: large_string, upper_bound: large_string> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: large_string
      child 5, upper_bound: large_string
  child 2, string_long: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: large_string, upper_bound: large_string> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: large_string
      child 5, upper_bound: large_string
  child 3, int: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: int32, upper_bound: int32> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: int32
      child 5, upper_bound: int32
  child 4, long: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: int64, upper_bound: int64> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: int64
      child 5, upper_bound: int64
  child 5, float: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: float, upper_bound: float> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: float
      child 5, upper_bound: float
  child 6, double: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: double, upper_bound: double> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: double
      child 5, upper_bound: double
  child 7, timestamp: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: timestamp[us], upper_bound: timestamp[us]> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: timestamp[us]
      child 5, upper_bound: timestamp[us]
  child 8, timestamptz: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: timestamp[us, tz=UTC], upper_bound: timestamp[us, tz=UTC]> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: timestamp[us, tz=UTC]
      child 5, upper_bound: timestamp[us, tz=UTC]
  child 9, date: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: date32[day], upper_bound: date32[day]> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: date32[day]
      child 5, upper_bound: date32[day]
  child 10, binary: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: large_binary, upper_bound: large_binary> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: large_binary
      child 5, upper_bound: large_binary
  child 11, fixed: struct<column_size: int64, value_count: int64, null_value_count: int64, nan_value_count: int64, lower_bound: fixed_size_binary[16], upper_bound: fixed_size_binary[16]> not null
      child 0, column_size: int64
      child 1, value_count: int64
      child 2, null_value_count: int64
      child 3, nan_value_count: int64
      child 4, lower_bound: fixed_size_binary[16]
      child 5, upper_bound: fixed_size_binary[16]
(Pdb)
```

Reply:
notice the `file_format: dictionary` in the 3rd row. And a few other differences

Reply:
@kevinjqliu thanks for the catch. Similar discrepancies exist in ```refs``` and ```entries```, I will make those changes in a separate PR.